.catalog-onboard-pipeline.yaml

```diff
@@ -6,7 +6,15 @@ offerings:
     catalog_id: 7df1e4ca-d54c-4fd0-82ce-3d13247308cd
     offering_id: 5fdd0045-30fc-4013-a8bc-6db9d5447a52
     variations:
-      - name: Code-Engine-d7b9
+      - name: basic
```

Comment:
will changing the variation name have any impact on catalog?

Reply:
We are not supporting an upgrade path from the 1.x version to 2.x. So we do not need to worry about changing variation names and any impact it will have